elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -154,9 +154,10 @@ See mixin blocks before selectors that contain the selector name and the three l
     box-shadow: inset 0 4px 0 0 #e00;
   }
 
-  &:before {
-    margin: 0 0 4px;
-  }
+  //@todo: decide if this is needed with pfe-icons in place
+  // &:before {
+  //   margin: 0 0 4px;
+  // }
```

Comment:
Shouldn't be, I definitely added it for the icon placeholder.

Reply:
Sounds good, I'll get it removed.